uswds/uswds-site
# USWDS-Site: Phase two compile updates

## Summary
Restructured the content in Phase 2 of the Getting Started for Developers guide to add clarity. 

## Related issue
Closes https://github.com/uswds/uswds-site/issues/1708

Recommend releasing this update with the settings page update in PR https://github.com/uswds/uswds-site/pull/1755

## Preview link
Preview link: [Phase 2 Compile ](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/al-add-table-context/documentation/getting-started/developers/phase-two-compile/)

## Problem statement
The guide for compiling USWDS Sass should give clear instructions for both users who want to use `uswds-compile` to create their Sass, and those who want to manage their own Sass compilation. However, the current page creates a single set of steps for both processes. Additionally, the instructions for `uswds-compile` can be confusing if the user is not very familiar with Sass compilation.

## Solution
By separating out the sections and adding clear headers, we can help the user understand which steps they need to take for their project. Additionally, adding more information about setting up the `gulpfile.js` for `uswds-compile` will help give users a clear step-by-step process to get up and running. 

## Testing and review
For copyeditors:
- Proofread for errors and clarity
- Confirm that the page structure makes sense and is easy to follow 

For developers:
- Confirm that all tasks and items listed are accurate and comprehensive
- Confirm that the page structure makes sense and is easy to follow
